doctor/lib/utils.py

```diff
@@ -354,3 +356,127 @@ def make_page_with_text(page, data, h, w):
     can.save()
     packet.seek(0)
     return packet
+
+
+def skew(obj):
+    """"""
+    if (matrix := obj.get("matrix")) is None:
+        return True
+
+    # Remove Skew
+    my_char_ctm = CTM(*matrix)
+    if my_char_ctm.skew_x != 0:
+        return False
+    return True
```

Comment:
The one instances where ive found skewed text in a PDF that is not a scan is either in the margin - or in a circular stamp used mostly in California - with the court - the judge name and the approved - denied text.

Reply:
In those instances it simply just removes the text - because it is usually appended to the original document or would obfuscate the content of the pdf.  I think.